README.md

```diff
@@ -2,45 +2,162 @@
+
+```CMake
+find_package(azure-storage-blobs-cpp CONFIG REQUIRED)
+target_link_libraries(myProject PRIVATE Azure::azure-storage-blobs)
+```
+
+> NOTE: All the Azure client libraries take a dependency on `azure-core-cpp` which provides functionality commonly needed by all Azure clients. When you install any client library via vcpkg, it will bring in all the necessary dependencies as well. You don't need to install those individually to get started.
 
-### Client: New Releases
+You can reference this [vcpkg Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) for more details.
 
-New wave of packages that we are announcing as **GA** and several that are currently releasing in **beta**. These libraries follow the [Azure SDK Design Guidelines for C++](https://azure.github.io/azure-sdk/cpp_introduction.html) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure::Core](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/core/azure-core/README.md).
+### Building your Application
 
-These new client libraries can be identified by the naming used for their folder, package, and namespace. Each will start with `azure`, followed by the service category, and then the name of the service. For example `azure-storage-blobs`.
+In order to use the SDK installed via vcpkg with CMake, you can use the toolchain file from vcpkg:
+
+```cmd
+> cmake -B [build directory] -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg repo]/scripts/buildsystems/vcpkg.cmake
+> cmake --build [build directory]
+```
+
+The **entry point** for most scenarios when using the SDK will be a top-level client type corresponding to the Azure service you want to talk to. For example, sending requests to blob storage can be done via the `Azure::Storage::Blobs::BlobClient` API.
+
+All the Azure C++ SDK headers needed to be included are located within the `<azure>` folder, with sub-folders corresponding to each service. Similarly, all types and APIs can be found within the `Azure::` namespace. For example, to use functionality form `Azure::Core`, include the following header at the beginning of your application `#include <azure/core.hpp>`.
+
+Here's an example application to help you get started:
+
+```C++
+#include <azure/core.hpp>
+#include <azure/storage/blobs.hpp>
+
+using namespace Azure::Storage::Blobs;
+
+int main()
+{
+	std::string containerName = "testcontainer";
+	std::string blobName = "sample-blob";
+
+	try
+	{
+		auto client = BlobClient::CreateFromConnectionString(
+            GetConnectionString(), containerName, blobName);
+	}
+	catch (Azure::Core::RequestFailedException& e)
+	{
+		puts(e.what());
+		return 1;
+	}
+	return 0;
+}
+```

```

Comment:
I avoided including this in https://github.com/Azure/azure-sdk-for-cpp/tree/UpdatedReadMe since it reveals a bit too much that can just be copy/pasted.